pypiserver/core.py

```diff
@@ -256,6 +256,22 @@ def _listdir(root):
                               relfn=fn[len(root) + 1:])
 
 
+def read_lines(filename):
+    """
+    Read the contents of `filename`, stripping empty lines and '#'-comments.
+    Return a list of strings, containing the lines of the file.
+    """
+    lines = []
+
+    try:
+        with open(filename) as f:
+            lines = [line.strip() for line in f.readlines() if line.strip() and not line.strip().startswith('#')]
```

Comment:
nit: we can avoid calling `strip()` so many times:

```suggestion
            lines = [
                line 
                for line in (ln.strip() for ln in f.readlines()) 
                if line and not line.startswith('#')
            ]
```

Reply:
:+1: